pkg/operator/connectivitycheckcontroller/Makefile

```diff
@@ -0,0 +1,21 @@
+# sync the podnetworkconnectivitychecks crd from github.com/openshift/api for embedding
```

Comment:
It's a weird place to put a Makefile. Do we really need it?

Reply:
Refactored out of the main `Makefile` to make the main `Makefile` more readable. Not a requirement.